sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -128,6 +128,54 @@ def __init__(self, body, **kwargs):
     def __str__(self):
         return str(self.message)
 
+    def __repr__(self):
+        # type: () -> str
+        # pylint: disable=bare-except
+        message_repr = "body={}".format(
+            str(self.body)
+        )
+        try:
+            message_repr += ", application_properties={}".format(self.application_properties)
+        except:
+            pass
+        try:
+            message_repr += ", session_id={}".format(self.session_id)
+        except:
+            pass
+        try:
+            message_repr += ", message_id={}".format(self.message_id)
+        except:
+            pass
+        try:
+            message_repr += ", content_type={}".format(self.content_type)
+        except:
+            pass
+        try:
+            message_repr += ", correlation_id={}".format(self.correlation_id)
+        except:
+            pass
+        try:
+            message_repr += ", to={}".format(self.to)
+        except:
+            pass
+        try:
+            message_repr += ", reply_to={}".format(self.reply_to)
+        except:
+            pass
+        try:
+            message_repr += ", reply_to_session_id={}".format(self.reply_to_session_id)
+        except:
+            pass
+        try:
+            message_repr += ", subject={}".format(self.subject)
+        except:
+            pass
+        try:
+            message_repr += ", scheduled_enqueue_time_utc={}".format(self.scheduled_enqueue_time_utc)
+        except:
+            pass
```

Comment:
in eventhub we have something like `event_repr += ", properties=<read-error>"` in the except clause, I think we should align with that?

e.g. https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/azure/eventhub/_common.py#L115

Reply:
I really don't think we should align with it - given repr is 1024 chars long only - i don't think <read-error> gives out any  information important enough

Reply:
so I was checking the [python official doc for `__repr__`](https://docs.python.org/3/reference/datamodel.html#object.__repr__) which mentioned that

> This is typically used for debugging, so it is important that the representation is information-rich and unambiguous.

So I thought `properties=<read-error>` is kinda important information for debugging process.

but I won't be bikeshedding on this, we could always add later

Reply:
```This is typically used for debugging, so it is important that the representation is information-rich and unambiguous.```

hmm, in this case it makes sense - adding them